refinery/file_store/tests.py

```diff
@@ -254,39 +213,53 @@ def setUp(self):
+    def test_translate_from_url_with_source_map(self):
+        with override_settings(
+                REFINERY_FILE_SOURCE_MAP={self.url_prefix: '/new/path/'}
+        ):
+            translate_file_source = generate_file_source_translator()
+        self.assertEqual(translate_file_source(self.url_source),
+                         self.url_source)
 
     def test_translate_from_url(self):
         translate_file_source = generate_file_source_translator()
-        source = translate_file_source(self.url_source)
-        self.assertEqual(source, self.url_source)
+        self.assertEqual(translate_file_source(self.url_source),
+                         self.url_source)
 
     def test_translate_from_absolute_path(self):
         translate_file_source = generate_file_source_translator()
-        source = translate_file_source(self.abs_path_source)
-        self.assertEqual(source, self.abs_path_source)
+        self.assertEqual(translate_file_source(self.abs_path_source),
+                         self.abs_path_source)
 
     def test_translate_from_relative_path_with_base_bath(self):
-        translate_file_source = \
-            generate_file_source_translator(base_path=self.base_path)
-        source = translate_file_source(self.rel_path_source)
-        self.assertEqual(source,
+        translate_file_source = generate_file_source_translator(
+            base_path=self.base_path
+        )
+        self.assertEqual(translate_file_source(self.rel_path_source),
                          os.path.join(self.base_path, self.rel_path_source))
 
-    def test_translate_from_relative_path_without_base_path(self):
-        translate_file_source = \
-            generate_file_source_translator(username=self.username)
-        source = translate_file_source(self.rel_path_source)
-        self.assertEqual(source, os.path.join(
-            settings.REFINERY_DATA_IMPORT_DIR,
-            self.username,
-            self.rel_path_source))
+    def test_translate_from_relative_path_with_username(self):
+        translate_file_source = generate_file_source_translator(
+            username=self.username
+        )
+        self.assertEqual(translate_file_source(self.rel_path_source),
+                         os.path.join(settings.REFINERY_DATA_IMPORT_DIR,
+                                      self.username, self.rel_path_source))
+
+    @override_settings(UPLOAD_BUCKET='refinery-upload')
+    def test_translate_from_relative_path_with_cognito_identity_id(self):
+        identity_id = 'us-east-1:{}'.format(uuid.uuid4())
+        translate_file_source = generate_file_source_translator(
+            identity_id=identity_id
+        )
+        self.assertEqual(
+            translate_file_source(self.rel_path_source),
+            "s3://{}/{}/{}".format(
+                settings.UPLOAD_BUCKET, identity_id, self.rel_path_source
+            )
+        )
 
-    def test_translate_from_relative_path_without_username_or_base_path(self):
```

Comment:
prefer this name

Reply:
Changed the name to make it more explicit.